src/debugAdapter/goDebug.ts

```diff
@@ -246,7 +263,22 @@ class Delve {
 	}
 
 	close() {
-		this.debugProcess.kill();
+        if (this.debugProcess === null) {                                                    
+            this.call<DebuggerState>('Command', [{ name: 'halt' }], (err, state) => {
+                if (err) {
+                    console.error('Failed to halt.');
+                }
+                // else TODO:  Can't find a way to kill remote delve.
+                // {
+                //     this.call<number>('Detach', [true], (err, ret) => {
+                //         if (err) {
+                //             console.error('Failed to detach.');
+                //         }
+            	// 	});                                                                                           
+                // }
+    		});                                               
+        }
+		else this.debugProcess.kill();
```

Comment:
Curly braces around this and similar in line with the rest of the formatting here.

Also - run `npm run lint` to see lint failures generally - they will fail the travis build as well.